src/applications/mhv-secure-messaging/util/constants.js

```diff
@@ -151,8 +151,15 @@ export const Alerts = {
   Message: {
     BLOCKED_MESSAGE_ERROR:
       'You are blocked from sending messages to this recipient.',
-    CANNOT_REPLY_BODY:
-      "The last message in this conversation is more than 45 days old. If you want to continue this conversation, you'll need to start a new message.",
+    CANNOT_REPLY_BODY: {
+      MAIN: 'The last message in this conversation is more than 45 days old.',
+      VISTA:
+        "If you want to continue this conversation, you'll need to start a new message.",
+      OH_PILOT:
```

Comment:
we can keep naming convention here as OH, not OH_PILOT as at some point pilot will no longer be a pilot